doc/changelogs/CHANGELOG_V10.md

```diff
@@ -59,6 +60,98 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="10.20.0"></a>
+## 2020-04-07, Version 10.20.0 'Dubnium' (LTS), @BethGriggs
+
+### macOS package notarization and a change in builder configuration
+
+The macOS binaries for this release, and future 10.x releases, are now
+being compiled on macOS 10.15 (Catalina) with Xcode 11 to support
+package notarization, a requirement for installing on .pkg files on
+macOS 10.15 and later. Previous builds of Node.js 10.x were compiled on
+macOS 10.10 (Yosemite). As binaries are still being compiled to support
+a minimum of macOS 10.10 (Yosemite) we do not anticipate this having a
```

Comment:
Minimum deployment target for Node.js 10 is maOS 10.7 (Lion): https://github.com/nodejs/node/blob/017909b847851f40613994506785273a951f9235/common.gypi#L470

Reply:
Thank you for linking the file, I was trying to find where that was defined earlier